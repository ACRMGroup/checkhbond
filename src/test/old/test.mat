residue ASN
donate	      25	      30	      31	     3
donate	      25	      34	      29	     1
donate	      25	      34	      31	     2
donate	      25	      35	      30	     1
donate	      25	      35	      31	     1
donate	      26	      33	      27	     1
donate	      26	      34	      32	     1
donate	      26	      35	      30	     1
donate	      29	      34	      26	     1
donate	      29	      34	      34	     2
donate	      30	      32	      34	     2
donate	      30	      33	      34	     1
donate	      30	      34	      34	     1
donate	      30	      35	      33	     1
donate	      30	      36	      31	     2
donate	      30	      36	      32	     2
donate	      30	      37	      30	     2
donate	      31	      34	      33	     1
donate	      31	      36	      32	     1
partnertodonate	      26	      28	      34	     1
partnertodonate	      26	      29	      35	     1
partnertodonate	      27	      28	      35	     1
partnertodonate	      32	      37	      27	     1
accept	      24	      34	      30	     1
accept	      25	      30	      30	     2
accept	      25	      30	      31	     1
accept	      25	      31	      30	     1
accept	      25	      31	      31	     1
accept	      25	      33	      29	     1
accept	      25	      34	      29	     1
accept	      26	      32	      32	     1
accept	      27	      33	      34	     1
accept	      27	      35	      33	     1
accept	      29	      32	      34	     1
accept	      29	      33	      34	     2
accept	      30	      33	      34	     2
accept	      30	      34	      27	     1
accept	      30	      34	      34	     2
accept	      30	      36	      31	     1
accept	      30	      36	      32	     2
accept	      30	      37	      30	     1
accept	      31	      35	      32	     1
accept	      31	      35	      33	     1
accept	      31	      36	      31	     1
accept	      32	      35	      28	     1
partnertoaccept	      19	      29	      30	     1
donate	      25	      30	      31	     3
donate	      25	      34	      29	     1
donate	      25	      34	      31	     2
donate	      25	      35	      30	     1
donate	      25	      35	      31	     1
donate	      26	      33	      27	     1
donate	      26	      34	      32	     1
donate	      26	      35	      30	     1
donate	      29	      34	      26	     1
donate	      29	      34	      34	     2
donate	      30	      32	      34	     2
donate	      30	      33	      34	     1
donate	      30	      34	      34	     1
donate	      30	      35	      33	     1
donate	      30	      36	      31	     2
donate	      30	      36	      32	     2
donate	      30	      37	      30	     2
donate	      31	      34	      33	     1
donate	      31	      36	      32	     1
partnertodonate	      26	      28	      34	     1
partnertodonate	      26	      29	      35	     1
partnertodonate	      27	      28	      35	     1
partnertodonate	      32	      37	      27	     1
accept	      24	      34	      30	     1
accept	      25	      30	      30	     2
accept	      25	      30	      31	     1
accept	      25	      31	      30	     1
accept	      25	      31	      31	     1
accept	      25	      33	      29	     1
accept	      25	      34	      29	     1
accept	      26	      32	      32	     1
accept	      27	      33	      34	     1
accept	      27	      35	      33	     1
accept	      29	      32	      34	     1
accept	      29	      33	      34	     2
accept	      30	      33	      34	     2
accept	      30	      34	      27	     1
accept	      30	      34	      34	     2
accept	      30	      36	      31	     1
accept	      30	      36	      32	     2
accept	      30	      37	      30	     1
accept	      31	      35	      32	     1
accept	      31	      35	      33	     1
accept	      31	      36	      31	     1
accept	      32	      35	      28	     1
partnertoaccept	      19	      29	      30	     1
residue TYR
donate	      18	      30	      31	     1
donate	      18	      31	      28	     1
donate	      18	      32	      27	     1
donate	      18	      33	      30	     1
donate	      18	      34	      29	     1
donate	      18	      34	      30	     2
donate	      19	      33	      30	     1
donate	      19	      35	      28	     1
donate	      30	      36	      40	     1
donate	      30	      37	      39	     1
donate	      30	      37	      40	     2
donate	      30	      38	      39	     4
donate	      31	      37	      39	     1
donate	      31	      37	      40	     1
donate	      31	      39	      39	     1
donate	      32	      37	      39	     1
donate	      32	      39	      38	     1
donate	      33	      38	      39	     1
donate	      34	      38	      38	     1
partnertodonate	      26	      39	      42	     1
partnertodonate	      27	      38	      43	     1
partnertodonate	      28	      40	      41	     1
accept	      18	      30	      31	     1
accept	      18	      31	      28	     1
accept	      18	      32	      27	     1
accept	      18	      33	      30	     1
accept	      18	      34	      29	     1
accept	      18	      34	      30	     2
accept	      19	      33	      30	     1
accept	      19	      35	      28	     1
accept	      30	      36	      40	     1
accept	      30	      37	      39	     1
accept	      30	      37	      40	     2
accept	      30	      38	      39	     4
accept	      31	      37	      39	     1
accept	      31	      37	      40	     1
accept	      31	      39	      39	     1
accept	      32	      37	      39	     1
accept	      32	      39	      38	     1
accept	      33	      38	      39	     1
accept	      34	      38	      38	     1
partnertoaccept	      15	      31	      33	     1
partnertoaccept	      16	      31	      33	     1
partnertoaccept	      16	      33	      31	     1
partnertoaccept	      26	      39	      42	     1
partnertoaccept	      26	      41	      39	     1
partnertoaccept	      27	      38	      43	     1
partnertoaccept	      28	      40	      41	     1
