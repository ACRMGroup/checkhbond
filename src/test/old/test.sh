../bin/retired/checkhbond_orig  -c 0.2 -m /acrm/usr/local/share/hb/data/hbmatrices.dat 1tsrB.pdb B131 B126 tyr
../bin/checkhbond -c 0.2 -m /acrm/usr/local/share/hb/data/hbmatrices.dat 1tsrB.pdb B131 B126 tyr
