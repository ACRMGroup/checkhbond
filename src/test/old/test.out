Pseudoenergy of best quality hydrogen bond: 5.877736
Pseudoenergy of best quality hydrogen bond: 5.877736
